sdk/dapp-kit/src/hooks/wallet/useWalletContext.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { WalletContext } from 'dapp-kit/src/contexts/WalletContext';
```

Comment:
I'd prefer to use relative imports everywhere across dapp-kit, does this work with our current setup?